### ISSUE TITLE
Feedback 'Color contrast'

### ISSUE THREAD

**Issue by dustinWeaver** [topcoderinc/va-kidney-ios#49]
### Message from Cyndi Anschutz Cooke sent on 2018-05-03 17:54:25 UTC###

> The icon and font color at the bottom of the main page does not have a high enough contrast with the background color. This poses an accessibility issue. See WCAG 2.0 guidelines for contrast ratios.

### Link to HockeyApp ###

* [https://rink.hockeyapp.net/manage/apps/727093/feedback/2143186](https://rink.hockeyapp.net/manage/apps/727093/feedback/2143186)
